modules/nf-core/president/main.nf

```diff
@@ -0,0 +1,63 @@
+process PRESIDENT {
+    tag "$fasta"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "docker://rkimf1/president:0.6.8--4616f6d"
+
+    input:
+    tuple val(meta), path(fasta)
+    path reference
+    val compress
+
+    output:
+    tuple val(meta), path("output/*.fasta*"), emit: fasta
+    tuple val(meta), path("output/*.tsv"), emit: report
+    tuple val(meta), path("output/*.log"), emit: log
+    path "versions.yml", emit: versions
```

Comment:
```suggestion
    tuple val(meta), path("output/*.fasta*"), emit: fasta
    tuple val(meta), path("output/*.tsv")   , emit: report
    tuple val(meta), path("output/*.log")   , emit: log
    path "versions.yml"                     , emit: versions
```

Reply:
fixed! is it ready to merge in your eyes? :)